examples/openshift/openshift-values.yaml

```diff
@@ -5,12 +5,17 @@ global:
       enabled: true
```

Comment:
Seems like all the changes in openshift-values.yaml are no longer needed? They were removed in the ingress controller type openshift pr.